packages/flutter/lib/src/material/text_field.dart

```diff
@@ -278,6 +279,11 @@ class TextField extends StatefulWidget {
   /// [Decoration.enabled] property.
   final bool enabled;
 
+  /// When this widget receives focus and is not completely visible (for example scrolled partially off the screen or overlapped by the keyboard)
```

Comment:
This doc comment needs to follow the style guide in [1], roughly:

```
<One sentence describing what the property is>
<empty line>
<more context info>
```

[1] https://www.dartlang.org/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary